lib/rules/no-unknown-property.js

```diff
@@ -193,7 +193,7 @@ function getStandardName(name) {
   if (SVGDOM_ATTRIBUTE_NAMES[name]) {
     return SVGDOM_ATTRIBUTE_NAMES[name];
   }
-  let i;
+  let i = -1;
```

Comment:
This seems to be useless, but I assume tells tsc what the type of `i` is. It could also be set to `0`, but `-1` is clearer semantically.

Reply:
If `i` is not initialized, it could potentially be `undefined`, that would make tsc reject `DOM_PROPERTY_NAMES[i]`.

Reply:
`[undefined]` on an object without that property is safely undefined, that's just the type system being overly strict :-)